woocommerce/woocommerce-android
# Dynamic dashboard/preparations

### Description
As agreed on Slack, this PR adds a copy of the my store screen, it's a copy of all of the package, which would make the development on the new screen less disruptive.

I didn't duplicate tests, as I think we'll need new unit tests for most of the changes, and where the old tests are usable, we'll move them when needed before doing any code cleanup at the end of the project.

### Testing instructions
Just a non-regression test, disable the feature flag, and confirm the app behaves correctly (this should be covered by UI tests, but just to be sure).
